src/style.css

```diff
@@ -1 +1,8 @@
 /* styles go here */
+.form {
+  margin-bottom: 10px;
+}
+
+fieldset {
```

Comment:
Dont style using tags, add propper class name and style using it

Reply:
I have corrected my code, please check now